### PR TITLE
DOC: add link to Vega dataset preview notebook in readme

### LINKED ISSUE #122: add vega-datasets JS notebook to docs ...
@domoritz sounds good! I'll do one better for you guys: 

I woke up this morning thinking we could use a simple vega datasets preview js notebook :)

https://observablehq.com/@randomfractals/vega-datasets

vega-datasets-notebook

I'll leave it up to you guys if you want to add this vega datasets preview utility Observable notebook to your editor or this datasets repo readme.md. 

This notebook can be used as a supplemental tool for online vega editor and examples that use these data sources since it's much faster in data loading and scrolling than what github and vega editor provides.

I might add something similar to the https://github.com/RandomFractals/vscode-vega-viewer as a split panel in vega chart preview in the next major release.

 cc @kanitw @arvind & @jheer 

Cheers! 🤗

_Originally posted by @RandomFractals in https://github.com/vega/vega-datasets/issues/64#issuecomment-489421032_

### PR DESCRIPTION
**Pull request by eitanlees**
closes #122

### CODE REVIEW

**Comment by RandomFractals** [comment]
@eitanlees this looks good! Thanks for adding it to docs.

**Comment by domoritz** [comment]
Thank you. @RandomFractals Can you update the notebook to use https://www.npmjs.com/package/vega-datasets. It has both a list of all datasets and links to the datasets/the data itself. The advantage of using this library would be that you don't have to manually maintain the list of datasets.

**Comment by RandomFractals** [comment]
sure thing! I can tend to it tomorrow. Can you still bless this PR?

Thanks!

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The list of sources is in [sources.md](https://github.com/vega/vega-datasets/blo
 
 To access the data in Observable, you can import `vega-dataset`. Try our [example notebook](https://beta.observablehq.com/@domoritz/vega-datasets). To access these datasets from Python, you can use the [Vega datasets python package](https://github.com/jakevdp/vega_datasets). To access them from Julia, you can use the [VegaDatasets.jl julia package](https://github.com/davidanthoff/VegaDatasets.jl).
 
+The [vega datasets preview notebook](https://observablehq.com/@randomfractals/vega-datasets) offers a quick way to browse the content of the available datasets.
```
Comment (domoritz) [inline README.md]:
```suggestion
The [Vega datasets preview notebook](https://observablehq.com/@randomfractals/vega-datasets) offers a quick way to browse the content of the available datasets.
```

**Pull request merged**